JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -49,8 +50,9 @@
     createLazyMethod(start + trackEvent);
     createLazyMethod(stop + trackEvent);
 
-    createLazyMethod(start + trackPage);
-    createLazyMethod(stop + trackPage);
+    //Record the current time for the methods startTrackPage and stopTrackPage
+    createLazyMethod(start + trackPage, new Date());
```

Comment:
Can the `new Date()` be calculated inside the method itself if no value was passed to it?